docker/compose/local-filer-backup-compose.yml

```diff
@@ -34,7 +34,7 @@ services:
       WEED_SINK_S3_DIRECTORY: "/"
       WEED_SINK_S3_AWS_ACCESS_KEY_ID: "some_access_key1"
       WEED_SINK_S3_AWS_SECRET_ACCESS_KEY: "some_secret_key1"
-      WEED_SINK_S3_S3_DISABLE_CONTENT_MD5_VALIDATIONE: "false"
+      WEED_SINK_S3_S3_DISABLE_CONTENT_MD5_VALIDATION: "false"
       WEED_SINK_S3_UPLOADER_PART_SIZE: "5242880"
```

Comment:
`WEED_SINK_S3_UPLOADER_PART_SIZE` should be changing to MB

Reply:
@chrislusf  fixed